### PR TITLE
Add WMCB tests for 4.6 community branch 

### PR DESCRIPTION
**Pull request by VaishnaviHire**
This PR adds test e2e tests for `community-4.6` branch
The tests run on OKD cluster

Ran `make jobs` and `make ci-operator-config`

Requires #14632

### CODE REVIEW

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:10]: I think should define the golang image within the repo [like we do for WMCO](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master.yaml#L7)

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml
Comment (sebsoto) [inline ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:33]: What the purpose of `--warn-undefined-variables` in this case?

Comment (sebsoto) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:33]: Applies to `make verify-all` as well

**Review by sebsoto — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml
Comment (VaishnaviHire) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:33]: I am not sure about a particular case. Keeping it consistent with other config [files](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-master.yaml#L41)

Comment (ravisantoshgudimetla) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:33]: I think it's part of make, please look at https://www.oreilly.com/library/view/managing-projects-with/0596006101/ch12.html

**Review by ravisantoshgudimetla — COMMENTED** [review]
Thank you for working on this @VaishnaviHire 

Mostly LGTM. Can you answer my question?

File: ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6-postsubmits.yaml
Comment (VaishnaviHire) [reply ci-operator/jobs/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6-postsubmits.yaml:1]: Yes by using the command `make jobs`

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml
Comment (VaishnaviHire) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml]: Should a separate PR be open to reflect this change in other branches?

Comment (aravindhp) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml]: A change like this can be a follow up PR. My suggestion would be to not make this change in this PR when enabling testing for the community branch.

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml
Comment (aravindhp) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:30]: I don't think @sebsoto's question is about what `--warn-undefined-variables` does but **why** is it being added here. And @VaishnaviHire's answer is consistency. I vaguely remember that `--warn-undefined-variables` was not added by us but by the release team and @VaishnaviHire is just following that pattern.

File: ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml
Comment (VaishnaviHire) [reply ci-operator/config/openshift/windows-machine-config-bootstrapper/openshift-windows-machine-config-bootstrapper-community-4.6__okd.yaml:10]: Will open up a follow-up PR once this merges

**Review by ravisantoshgudimetla — APPROVED** [review]
/lgtm

Thank you for working on this @VaishnaviHire

**Pull request merged**
